### ISSUE TITLE
Syntaxnet - Spanish test

### ISSUE THREAD

**Issue by jag2kn** [tensorflow/models#178]
https://github.com/tensorflow/models/tree/master/syntaxnet

Hi, how can I do test running syntaxnet in Spanish text?

In the paper "Globally Normalized Transition-Based Neural Networks" you run the CoNLL2009 test in Spanish and get 92.62%, how can reproduce this experiment?

And how can test this with new sentences?

Thanks you very much :)

**Comment by jag2kn** [comment]
This is the data?

http://ufal.mff.cuni.cz/conll2009-st/train-dev-data.html
http://ufal.mff.cuni.cz/conll2009-st/eval-data.html

**Comment by jag2kn** [comment]
http://universaldependencies.org/docs/format.html

**Comment by jag2kn** [comment]
:(

**Comment by calberti** [comment]
Slav probably closed this because he saw that you already found the data.  You should be able to follow the README to train a model on the data you point to above (ie. the conll2009-st links).  You will get a reasonable model, but you will probably not match the numbers in the paper.  To get the 92.62% UAS however you would need to model morphology, add more features, and run a grid search on the parameters.  Morphology and the additional features are not supported yet in SyntaxNet.

**Comment by jag2kn** [comment]
Hi Calberti yes and thanks for the reply.

These are the data, but my question is about "how I run these tests in Spanish?".

I know that the results will probably not match. But I have no problem with this, I want to reproduce the test in the Spanish corpus
